README.md

```diff
@@ -34,3 +34,45 @@ This plugin allows for the creation and management of Mattermost kubernetes inst
   | Mattermost E20 License | `E20 license file contents` |
 8. Enable the plugin
 The plugin is now usable. Make changes to your plugin and run `make deploy` again to run those changes on your local Mattermost server.
+
+
+## Release
+
+To release a new version of the `cloud plugin` please do the following steps:
+
+- Bump the version to the desire version in the files:
+  - `plugin.json`
```

Comment:
You only need to change it here then run `make apply`

Reply:
done